src/rebalancers/libraries/SlipstreamLogic.sol

```diff
@@ -0,0 +1,61 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+import { ICLPool } from "../interfaces/ICLPool.sol";
+import { ICLPositionManager } from "../interfaces/ICLPositionManager.sol";
+import { PoolAddress } from "./slipstream/PoolAddress.sol";
+import { Rebalancer } from "../Rebalancer.sol";
+
+library SlipstreamLogic {
+    // The Slipstream Factory contract.
+    address internal constant CL_FACTORY = 0x5e7BB104d84c7CB9B682AaC2F3d509f5F406809A;
+
+    // The Slipstream Pool Implementation contract.
+    address internal constant POOL_IMPLEMENTATION = 0xF926b5acC092E396A3b337642Be2E4cAe3f5da8E;
```

Comment:
Don't think it changes a lot but the last deployed one and listed in Aerodrome Docs is: 0xeC8E5342B19977B4eF8892e02D8DAEcfa1315831